src/Sarif/Converters/ClangAnalyzerConverter.cs

```diff
@@ -355,7 +354,7 @@ private void ReadPlistDictionary(XmlReader xmlReader, IList<Result> results)
                                     throw new InvalidDataException("Expected key value before dictionary data.");
                                 }
 
-                                string value = xmlReader.ReadElementContentAsString();
```

Comment:
See, here's an example where you removed the variable but kept the call, which makes sense.